src/Appwrite/Platform/Workers/Certificates.php

```diff
@@ -171,10 +171,7 @@ private function execute(Domain $domain, Database $dbForPlatform, Mail $queueFor
             $certificate->setAttribute('issueDate', DateTime::now());
             $success = true;
         } catch (Throwable $e) {
-            $logs = $e->getMessage();
-
-            // Set exception as log in certificate document
-            $certificate->setAttribute('logs', \mb_strcut($logs, 0, 1000000));// Limit to 1MB
+            $certificate->setAttribute('logs', 'Failed to create custom domain');
```

Comment:
1. this isn't always for custom domain.
2. we should still capture logs to help us investigate the issue.

Reply:
1. Cool, Could you phrase it in a more general way?
2. For that we have Sentry, no?